xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -218,11 +218,11 @@
   
  Any callbacks or cancelable operations registered with the <xref:System.Threading.CancellationToken> will be executed.  
   
- Cancelable operations and callbacks registered with the token should not throw exceptions.  
+ It is recommended that cancelable operations and callbacks registered with <xref:System.Threading.CancellationToken> do not throw exceptions. However, this overload of Cancel will aggregate any exceptions thrown into an <xref:System.AggregateException>, such that one callback throwing an exception will not prevent other registered callbacks from being executed. 
```

Comment:
Although there are numerous uses of "it is recommended that" in the documentation, it's a very awkward phrase. Better is, "We recommend that cancelable operations and callbacks registered with \<xref:...> not throw exceptions."